Reference/V9-Config/ContentSettings/index-v9.md

```diff
@@ -0,0 +1,195 @@
+- When using XPath, there is no "context" (i.e. you can't find the node based on "currentPage") so needs to be a global absolute path.
+
+:::
+
+:::warning Remember to recycle the app pool to make sure changes to this section take effect. :::
+
+If you have multiple sites, with different cultures, setup in your tree then you will need to setup the errors section like below:
+
+```json
+"Error404Collection": [
+  {
+    "ContentId": 1,
+    "Culture": "default"
+  },
+  {
+    "ContentId": 200,
+    "Culture": "en-US"
+  }]
+```
+
+If you have more than two sites and for some reason forget to update the above section with a 404 page and a culture then the default page will act as a fallback. Same happens if you for some reason forget to define a hostname on a site.
+
+### Preview badge
+
+This allows you to customize the preview badge being shown when you're previewing a node.
+
+```json
+"PreviewBadge": "<![CDATA[<a id=\"umbracoPreviewBadge\" style=\"position: absolute; top: 0; right: 0; border: 0; width: 149px; height: 149px; background: url('{1}/preview/previewModeBadge.png') no-repeat;\" href=\"{0}/endPreview.aspx?redir={2}\"><span style=\"display:none;\">In Preview Mode - click to end</span></a>]]>"
+```
+
+### Macro errors
+
+This setting allows toy to specify how errors in macros should be handled.
+
+Options:
+* Inline - Default Umbraco behaviour, show an inline error within the macro bu allow the page to continue rendering.
+* Silent - Silently suppress the error and do not display the offending macro.
+* Throw - Throw an exception.
+* Content - Silently suppress the error, and dusplay custom content reported in the error event args.
+
+### Disallowed upload files
+
+This setting consists of a list of file extensions that editors shouldn't be allowed to upload via the backoffice.
+
+### Allowed upload files
+
+If greater control is required than available from the above, this setting can be used to store a list of file extensions. If provided, only files with these extensions can be uploaded via the backoffice.
+
+### Show deprecated property editors
+
+This setting is used for controlling whether or not the data types marked as obsolete should be visible in the dropdown when creating new data types.
+
+By default this is set to false. To make the obsolete data types visible in the dropdown change the value to **true**
+
+### Login background image
+
+You can specify your own background image for the login screen here. The image will automatically get an overlay to match backoffice colors. This path is relative to the `wwwroot/umbraco` path. The default location is: `wwwroot/umbraco/assets/img/installer.jpg`.
+
+### Login logo image
+
+You can specify for own image for the login screen logo. This path is relative to the `wwwroot/umbraco` path. The default location is: `wwwroot/umbraco/assets/img/application/umbraco_logo_white.svg`.
```

Comment:
Not quite sure what the first sentence means here 🤔

Is there missing a word?

Reply:
Yeah I think I did an oopsie, I've tried updating it to something that hopefully makes a bit more sense 😄